FlowCrypt/Controllers/Setup/SetupGenerateKeyViewController.swift

```diff
@@ -85,7 +85,12 @@ extension SetupGenerateKeyViewController {
 
             let encryptedPrv = try self.core.generateKey(passphrase: passPhrase, variant: .curve25519, userIds: [userId])
 
-            try awaitPromise(self.backupService.backupToInbox(keys: [encryptedPrv.key], for: self.user))
+            let semaphore = DispatchSemaphore(value: 0)
```

Comment:
This method uses a lot of other methods and too big for the first step. So I just wait for async task.

Reply:
Ok. Can do in another PR.